### PR TITLE
update rari-capital solmate to just solmate

### PR DESCRIPTION
**Pull request by roninjin10**
- solmate is now under transmissions11 github and published as the name solmate to npm

### CODE REVIEW

**Comment by tynes** [comment]
I think this needs a rebase

**Comment by ajsutton** [comment]
ooo, that's a lot of changed files... Were they all meant to change or is there a meaningful change in there somewhere and a bunch of things that just got regenerated?

**Comment by roninjin10** [comment]
Looks like it's a breaking upgrade 
image

**Comment by roninjin10** [comment]
@ajsutton find and replace of an import from @rari-capital/solmate to solmate

**Comment by roninjin10** [comment]
I think the deployment files that got changed in the find-and-replace should be reverted?

File: packages/contracts-bedrock/deployments/goerli/L1CrossDomainMessenger.json
Comment (maurelian) [inline packages/contracts-bedrock/deployments/goerli/L1CrossDomainMessenger.json:1]: why would the goerli deployments have changed?

Comment (roninjin10) [reply packages/contracts-bedrock/deployments/goerli/L1CrossDomainMessenger.json:1]: It was find and replacing I think every deployment file that changed needs to get reverted

**Comment by roninjin10** [comment]
Gonna close this in favor of @tynes converting contracts package

**Pull request closed without merging**